chainer/functions/loss/ctc.py

```diff
@@ -217,6 +220,8 @@ def connectionist_temporal_classification(x, t, blank_symbol):
     <http://www.cs.toronto.edu/~graves/preprint.pdf>`_
 
     """
+    if not isinstance(x, collections.Iterable):
```

Comment:
Please change `collections.Iterable` to `collections.Sequence` as we access the element of `x` by indices.